spec/actionizer/inputs_spec.rb

```diff
@@ -51,8 +51,10 @@ def call; end
           def call; end
         end
       end
-      it 'raises an error' do
-        expect { failing_class.call(bar: 'oops') }.to raise_error(ArgumentError)
+      it 'fails gracefully and sets the error message' do
+        result = failing_class.call(bar: 'oops')
+        expect(result).to be_failure
+        expect(result.error).not_to be_nil
```

Comment:
Added explicit checking for error messages in https://github.com/mikenichols/actionizer/pull/17/commits/fa834e69b1a854445ce0df381ce751dce6369caf